packages/multi-trait-rubric/configure/src/__tests__/index.test.jsx

```diff
@@ -0,0 +1,530 @@
+                        'Errors in grammar, punctuation, capitalization and/or spelling are present and some distract from meaning',
+                        'Errors in grammar, punctuation, capitalization and/or spelling are present but don’t distract from meaning',
+                        'Few errors in grammar, punctuation,\n' + '\n' + 'capitalization and/or spelling',
+                    ],
+                },
+                {
+                    name: 'Voice',
+                    standards: [],
+                    description: "the personal tone and flavor of the author's message",
+                    scorePointsDescriptors: [
+                        'Student’s response is blank, not in English, not legible, or does not respond to the prompt.',
+                        'Not concerned with audience or purpose\n' +
+                        '\n' +
+                        'No viewpoint (perspective) used\n' +
+                        '\n' +
+                        'Writing is mechanical and lifeless',
+                        'Shows beginning awareness of audience/purpose\n' +
+                        '\n' +
+                        'Some viewpoint (perspective) used throughout the piece\n' +
+                        '\n' +
+                        'Writing is distant, too formal or informal',
+                        'Awareness of audience; purpose is clear most of the time\n' +
+                        '\n' +
+                        'Uses viewpoint (perspective) throughout most of the paper\n' +
+                        '\n' +
+                        'Writing is pleasant, agreeable and satisfying',
+                        'Powerful connection with audience; purpose is clearly communicated\n' +
+                        '\n' +
+                        'Maintains strong viewpoint (perspective) throughout entire piece\n' +
+                        '\n' +
+                        'Writing is expressive, engaging and has lots of energy',
+                    ],
+                },
+            ],
+        },
+        {
+            maxPoints: 2,
+            scorePointsLabels: ['Non-Scorable', 'Unsatisfactory', 'Satisfactory'],
+            traitLabel: 'Category',
+            traits: [
+                {
+                    name: 'Presentation',
+                    standards: [],
+                    description: '',
+                    scorePointsDescriptors: [
+                        'Handwriting is unreadable, or response is blank, not in English, or too brief to evaluate. ',
+                        'Handwriting poor\n' + '\n' + 'Overall appearance is distracting to unacceptable',
+                        'Handwriting is generally legible\n' + '\n' + 'Overall appearance is acceptable or better',
+                    ],
+                },
+            ],
+        },
+    ],
+    ...extras,
+});
+const configuration = () => ({
```

Comment:
Let's not copy defaults here. Instead, you can add a test that if you don't set any configuration, the defaults will be used & a test that if you add "SOME configuration properties, they will overwrite the defaults.